config/runtime.exs

```diff
@@ -40,7 +43,10 @@ else
     static_url: static_url
 end
 
-config :accent, Accent.Repo, url: System.get_env("DATABASE_URL") || "postgres://localhost/accent_development"
+config :accent, Accent.Repo,
+  pool_size: Utilities.string_to_integer(System.get_env("DATABASE_POOL_SIZE")),
```

Comment:
What is going to happen if the option is set to `nil` if the env var is not present?

Reply:
It will force Ecto to take its default value (10). This is how we do it in `elixir-boilerplate` too.